core/src/test/scala/org/apache/spark/h2o/H2OFrameToRDDTestSuite.scala

```diff
@@ -51,6 +51,28 @@ class H2ORDDTest extends FunSuite with SharedSparkTestContext {
     rdd.unpersist()
   }
 
+  test("RDD[IntHolder] with nulls, does it hold nulls?") {
+
+    def almostDefined(i: Long) = Some(i.toInt) filter (_ % 31 != 0)
+
+    val rdd = sc.parallelize(1 to 1000, 100).map( v => IntHolder(almostDefined(v)))
+    val h2oFrame:H2OFrame = hc.asH2OFrame(rdd)
+
+    assertInvariantsWithNulls(rdd, h2oFrame, (row, vec) => {
+      val row1 = row + 1
```

Comment:
Maybe rowIdx would be better ? When I read it for first time I thought `row` represents really a row